manifests/overlays/kubeflow-platform/kustomization.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kubeflow
+sortOptions:
+  order: fifo
+resources:
+  - ../../third-party/jobset # Comment this line if JobSet is installed on the Kubernetes cluster.
+  - ../../base/crds
+  - ../../base/runtimes
+  - ../../base/manager
+  - ../../base/rbac
+  - ../../base/webhook
+  - kubeflow-trainer-roles.yaml
+
+# Update the Kubeflow Trainer controller manager & runtimes image tag.
+images:
+  - name: kubeflow/trainer-controller-manager
```

Comment:
This should have correct image: https://github.com/kubeflow/trainer/blob/3658d1c0ee6bad1a7de3f15703b9d51807ed5920/manifests/overlays/manager/kustomization.yaml#L17